dashboard/config/routes.rb

```diff
@@ -209,10 +209,9 @@ def redirect_to_teacher_dashboard
   post '/admin/gatekeeper/set', :to => 'dynamic_config#gatekeeper_set', as: 'gatekeeper_set'
   get '/admin/:action', controller: 'reports', as: 'reports'
 
-  get '/stats/usage/:user_id', to: 'reports#usage', as: 'usage'
+  get '/stats/usage/:user_id', to: redirect_to_teacher_dashboard
   get '/stats/students', to: redirect_to_teacher_dashboard
-  get '/stats/:user_id', to: 'reports#user_stats', as: 'user_stats'
-  get '/stats/level/:level_id', to: 'reports#level_stats', as: 'level_stats'
+  get '/stats/:user_id', to: redirect_to_teacher_dashboard
```

Comment:
Is the intent to maintain these redirects forever? If not, consider adding a TODO to remove the redirect after some time.